src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.commands.Command;
+import duke.exceptions.DukeUnknownInputException;
+import duke.commands.*;
```

Comment:
Remember not to use wildcard imports. Explicitly say each package you import.
IntelliJ might have auto merged them, you can change this in the settings

Reply:
I see I think IntelliJ must have auto merged them. Thanks for letting me know.